Sources/SwiftPhoenixClient/Message.swift

```diff
@@ -66,16 +65,16 @@ public class Message {
     self.rawPayload = payload
     self.joinRef = joinRef
   }
-  
-  init?(json: [String: Any]) {
-    self.ref = json["ref"] as? String ?? ""
-    self.joinRef = json["join_ref"] as? String
+
+  init?(json: [Any?]) {
```

Comment:
Not sure if `[Any]` and `[Any?]` are the same in practice. Technically, the message array can contain nulls, `[null, null, "topic", "event", ["some":"payload"]]`, but both `[Any]` and `[Any?]` appear to work correctly.

Reply:
Let's leave it as `[Any?]` here for reminder that values can be `nil`